content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md

```diff
@@ -207,25 +222,46 @@ When starting the kubelet, if the file specified by `--kubeconfig` does not exis
 ```
 {{< /note >}}
 
-Additionally, in 1.7 the kubelet implements __alpha__ features for enabling rotation of both its client and/or serving certs.
-These can be enabled through the respective `RotateKubeletClientCertificate` and `RotateKubeletServerCertificate` feature
-flags on the kubelet, but may change in backward incompatible ways in future releases.
+Additionally, in 1.7 the kubelet implements __beta__ features for enabling
+rotation of both its client and/or serving certs.  These can be enabled through
+the respective `RotateKubeletClientCertificate` and
+`RotateKubeletServerCertificate` feature flags on the kubelet and are enabled by
+default.
+
+`RotateKubeletClientCertificate` causes the kubelet to rotate its client
+certificates by creating new CSRs as its existing credentials expire. To enable
+this feature pass the following flag to the kubelet:
 
 ```
---feature-gates=RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true
+--rotate-certificates
 ```
 
-`RotateKubeletClientCertificate` causes the kubelet to rotate its client certificates by creating new CSRs as its existing
-credentials expire. `RotateKubeletServerCertificate` causes the kubelet to both request a serving certificate after
-bootstrapping its client credentials and rotate the certificate. The serving cert currently does not request DNS or IP
-SANs.
+`RotateKubeletServerCertificate` causes the kubelet to both request a serving
+certificate after bootstrapping its client credentials and to rotate that
+certificate. To enable this feature pass the following flag to the kubelet:
+
+```
+--rotate-server-certificates
+```
+
+{{< note >}}
+**Note:** The CSR approving controllers implemented in core Kubernetes do not
+approve node serving certificates for [security
+reasons](https://github.com/kubernetes/community/pull/1982). To use
+`RotateKubeletServerCertificate` operators need to run a custom approving
+controller, or manually approve the serving certificate requests.
+{{< /note >}}
 
 ## kubectl approval
-The signing controller does not immediately sign all certificate requests. Instead, it waits until they have been flagged with an
-"Approved" status by an appropriately-privileged user. This is intended to eventually be an automated process handled by an external
-approval controller, but for the alpha version of the API it can be done manually by a cluster administrator using kubectl.
-An administrator can list CSRs with `kubectl get csr` and describe one in detail with `kubectl describe csr <name>`. Before the 1.6 release there were
-[no direct approve/deny commands](https://github.com/kubernetes/kubernetes/issues/30163) so an approver had to update
-the Status field directly ([rough how-to](https://github.com/gtank/csrctl)). Later versions of Kubernetes offer `kubectl certificate approve <name>` and `kubectl certificate deny <name>` commands.
+
+The signing controller does not immediately sign all certificate requests.
```

Comment:
This is a bit confusing. By default `kube-controller-manager` will auto-approve certs that pass SAR review.
This paragraph can be mis-read as: "you must either manually approve all CSRs or write a custom controller to do that".